src/HomePage/components/MovieCard/MovieCard.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import Popup from "../Popup/Popup";
+import './MovieCard.css'
+import '../Popup/Popup.css'
+
+interface MovieCardInterface {
+    title: string,
+    popularity: number
+}
+
+export function MovieCard(props: MovieCardInterface) {
+    const [isOpen, setIsOpen] = useState(false);
+    const [isEdited, setIsEdited] = useState(false);
+    const [isDeleted, setIsDeleted] = useState(false);
+
+    const togglePopup = () => {
+        setIsOpen(!isOpen);
+    }
+
+    const toggleEdit = () => {
+        setIsEdited(!isEdited);
+    }
+
+    const toggleDelete = () => {
+        setIsDeleted(!isDeleted);
+    }
+
+    return (<div className='card' onClick={togglePopup}>
+        <div>{props.title}</div>
+
+        <div>{props.popularity}</div>
+        {isOpen && <Popup
```

Comment:
As we discussed, you didn't have to do any handlers for this task. it would be enough to have buttons on this page without handling. But thumbs up to you for doing this ahead!